packages/perseus-editor/src/editor-page.tsx

```diff
@@ -83,15 +86,21 @@ class EditorPage extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
 
+        // Convert any widgets that need to be converted to newer widget types
+        let convertedQuestionJson: PerseusRenderer = props.question;
+        if (props.question) {
+            convertedQuestionJson = convertDeprecatedWidgets(props.question);
+        }
```

Comment:
Since `convertDeprecatedWidgets` takes null/undefined as a parameter option, couldn't this be simplified to:
```
const convertedQuestionJson = convertDeprecatedWidgets(props.question);
```

Reply:
Hmm open to feedback on this one! 

The approach I've taken has been to ensure we're only ever calling convertQuestionJson if we already have the data. Otherwise we need a TON of null checks across our code. I've updated convertDeprecatedWidgets to no longer return null